db/migrate/20151117153302_add_default_image_to_stickers.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultImageToStickers < ActiveRecord::Migration
+  def change
+    change_column_default(:stickers, :image_url, 'http://cumbrianrun.co.uk/wp-content/uploads/2014/02/default-placeholder.png')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.